scheduler/scheduler.go

```diff
@@ -148,6 +148,15 @@ func (s *Scheduler) Schedule(ctx context.Context) error {
 					"span":     time.Since(distroStartTime).String(),
 					"duration": time.Since(distroStartTime),
 				})
+				if len(d.runnableTasksForDistro) != len(res.taskQueueItem) {
+					grip.Alert(message.Fields{
```

Comment:
I'd do: `grip.AlertWhen( len(d.runnableTasksForDistro) != len(res.taskQueueItem), ` 

and omit the if block

Reply:
I left this in since I'm doing more with it now